### DISCUSSION TITLE
我是小白，大家能不能帮帮我谢谢

### DISCUSSION THREAD

**Discussion by JuzuSss** [aliencaocao/netease_cloudmusic_discord_rpc#30] [Q&A] [answered]
我下载build 和 requirement 的时候总是显示错误不知道为什么，在src 里面和外面都报错

**Comment by aliencaocao** [comment]
报错信息发一下
另外你如果想用的话直接去release下载exe运行就行

**Reply by JuzuSss** [reply]
![屏幕截图 2023-12-10 140448](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/99507916/96095e73-6b14-40f0-b65d-537191d144ad)
下载 release 后discord 上没有显示啊. 
下载requirement 的报错： 
PS C:\Users\Juzu-Blue> cd C:\Users\Juzu-Blue\Downloads\netease_cloudmusic_discord_rpc-master\netease_cloudmusic_discord_rpc-master\src
PS C:\Users\Juzu-Blue\Downloads\netease_cloudmusic_discord_rpc-master\netease_cloudmusic_discord_rpc-master\src> pip install -r requirements.txt
Collecting pyMeow@ https://github.com/qb-0/pyMeow/releases/download/1.53.36/pyMeow-1.53.36.zip (from -r requirements.txt (line 1))
  Downloading https://github.com/qb-0/pyMeow/releases/download/1.53.36/pyMeow-1.53.36.zip (2.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.5/2.5 MB 4.5 MB/s eta 0:00:00
ERROR: Could not install packages due to an OSError: [Errno 22] Invalid argument: 'C:\\Users\\Juzu-Blue\\AppData\\Local\\Temp\\pip-unpack-5qqr21bl\\pyMeow-1.53.36.zip'

**Reply by aliencaocao** [reply]
麻烦你仔细看一下readme，里面明确提到出现你这个错误改怎么做
见https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/issues/21
至于第二个问题，是你系统防火墙的问题，把安装文件给误杀了。但是如果你只想使用，直接用exe就好了

**Reply by JuzuSss** [reply]
![屏幕截图 2023-12-10 143119](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/99507916/2ade0c79-6058-42fc-9cd9-c93575ea7638) 有勾的 😭

**Reply by aliencaocao** [reply, marked as answer]
https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/releases/tag/0.3.2
试试最新版本

**Reply by aliencaocao** [reply]
但是你用的好像是3.0，这个暂时还在开发，目前不支持

**Reply by JuzuSss** [reply]
软件在运行，但是还是不显示

**Reply by aliencaocao** [reply]
因为你用的3.0测试版，现在不支持，readme里也写了

**Reply by JuzuSss** [reply]
下载了你刚发的3.2

**Reply by aliencaocao** [reply]
我是指你的网易云用的是3.0

**Reply by JuzuSss** [reply]
哦好的麻烦你了我试试下载另一个网易云版本看看谢谢

**Reply by JuzuSss** [reply]
好使，作者nb  👍👍👍

**Reply by aliencaocao** [reply]
感谢使用，还请加个星
